examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -96,6 +97,15 @@ Example of a secret can be found in [glusterfs-provisioning-secret.yaml](gluster
 
 * `gidMin` + `gidMax` : The minimum and maximum value of GID range for the storage class. A unique value (GID) in this range ( gidMin-gidMax ) will be used for dynamically provisioned volumes. These are optional values. If not specified, the volume will be provisioned with a value between 2000-2147483647 which are defaults for gidMin and gidMax respectively.
 
+* `volumetype` : The volume type and its parameters can be configured with this optional value. If the volume type is not mentioned, by default the provisioner will create a distribute replica 3 volume.
```

Comment:
A link to Gluster/Heketi docs about would be nice, people probably don't know what's "disperse volume".